pkgs/tools/misc/yubikey-manager-qt/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv
+, fetchurl
+, makeWrapper
+, pyotherside
+, pythonPackages
+, python3
+, qmake
+, qt5
+, yubikey-manager
+, yubikey-personalization
+}:
+
+with import <nixpkgs> {};
+
+let
+  qmlPath = qmlLib: "${qmlLib}/${qt5.qtbase.qtQmlPrefix}";
+
+  qml2ImportPath = lib.concatMapStringsSep ";" qmlPath [
+    qt5.qtbase.bin qt5.qtdeclarative.bin pyotherside qt5.qtquickcontrols qt5.qtquickcontrols2.bin qt5.qtgraphicaleffects
+  ];
+
+in stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "yubikey-manager-qt";
+  version = "1.1.0";
+
+  src = fetchurl {
+    url = "https://developers.yubico.com/yubikey-manager-qt/Releases/yubikey-manager-qt-${version}.tar.gz";
+    sha256 = "8049a233a8cca07543d745a9f619c0fc3afb324f5d0030b93f037b34ac1c5e66";
+  };
+
+  nativeBuildInputs = [ makeWrapper python3.pkgs.wrapPython qmake ];
+
+  sourceRoot = ".";
+
+  postPatch = ''
+    substituteInPlace ykman-gui/deployment.pri --replace '/usr/bin' "$out/bin"
+  '';
+
+  buildInputs = [ pythonPackages.python stdenv qt5.qtbase qt5.qtgraphicaleffects qt5.qtquickcontrols qt5.qtquickcontrols2 pyotherside ];
+
+  buildPhase = ''
+    qmake
+    make
```

Comment:
Manually invoking `qmake` and `make` shouldn't be needed.

Reply:
Oh, of course. That's just because I started from a "normal" `callPackage` until I found out about the `libsForQt5.callPackage` helper.